docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -10,113 +10,139 @@ keywords: changelog, table, materialize
 
 ## Context
 
-You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog stream into a table. This is broadly called *materializing* a changelog stream into a table.
+You have a topic or a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog into a table. This is broadly called *materializing* a changelog into a table.
 
-## In action
+## Materializing a changelog topic
 
-```sql
-CREATE TABLE t1 AS
-    SELECT k,
-           LATEST_BY_OFFSET(v1) AS v1,
-           LATEST_BY_OFFSET(v2) AS v2,
-           LATEST_BY_OFFSET(v3) AS v3
-    FROM s1
-    GROUP BY k
-    EMIT CHANGES;
-```
-
-## Materializing a changelog stream
-
-In ksqlDB, you derive new tables by aggregating other streams and tables. To create a table that reflects the latest values for each key, use the `LATEST_BY_OFFSET` aggregation.
-
-Begin by telling ksqlDB to start all queries from the earliest point in each topic.
+If you have a `changelog` topic, and you want a view of the data that reflects the latest values for each key then simply create a table with the `changelog` topic using the `CREATE SOURCE TABLE` statement.
 
+Let's say that you have the following data in your `changelog` topic where the first row is the record that has the earliest offset and the last row is the record that has the latest offset:
 ```sql
-SET 'auto.offset.reset' = 'earliest';
++------+------+------+------+
+|K     |V1    |V2    |V3    |
++------+------+------+------+
+|k1    |0     |a     |true  |
+|k2    |1     |b     |false |
+|k1    |2     |c     |false |
+|k3    |3     |d     |true  |
+|k2    |4     |e     |true  |
 ```
+Note that there are 2 records for the keys `k1` and `k2` in the `changelog` topic.
 
-Make a stream `s1` with four columns. `k` represents the key of the table. Rows with the same key represent information about the same entity. `v1`, `v2`, and `v3` are various value columns.
-
+Make a table `latest_view` with four columns. `k` represents the key of the table. Rows with the same key represent information about the same entity. `v1`, `v2`, and `v3` are various value columns.
 ```sql
-CREATE STREAM s1 (
-    k VARCHAR KEY,
+CREATE SOURCE TABLE latest_view (
```

Comment:
@spena I cannot remember but does `CST` respect `auto.offset.reset` config? If yet, we should add that it should be set to `"earliest"` -- if it ignores the config and always reads from earliest, we should also mention it IMHO.

Reply:
It respects that config. The initial idea was to read always from earliest, but we decided to respect it instead.